server/src/immich/main.ts

```diff
@@ -19,6 +20,7 @@ export async function bootstrap() {
   app.set('trust proxy', ['loopback', 'linklocal', 'uniquelocal']);
   app.set('etag', 'strong');
   app.use(cookieParser());
+  app.use(compression({ level: 1 }));
```

Comment:
Looks like Cloudflare found level 4 to be best for brotli and level 8 for gzip

https://blog.cloudflare.com/this-is-brotli-from-origin

Reply:
Ooh interesting, let me compare

Reply:
GZip level 8:

![gzip8](https://github.com/immich-app/immich/assets/101130780/977e4457-a0f2-460e-9809-b025a979badf)

GZip level 4:

![gzip4](https://github.com/immich-app/immich/assets/101130780/79a0f74f-fee0-4d44-ba82-a575663f3b4d)

GZip level 1:

![gzip1](https://github.com/immich-app/immich/assets/101130780/7fa91dbf-2c05-4280-974d-19ab32f779d2)

No compression:

![no-compression](https://github.com/immich-app/immich/assets/101130780/ad903481-931f-4594-81bf-7f637ab6226a)

Reply:
Looking at the time bucket response, it goes from 328kb to 46.5kb with level 1, but level 8 only lowers that by 3kb down to 43.6kb and ends up taking longer. Level 4 seems reasonable, but the sizes all look pretty similar to level 1 so hard to tell if it's better overall.